Hardware/Hardware/CardReader/CardReaderEvent.swift

```diff
@@ -21,12 +21,6 @@ public enum CardReaderEvent {
     /// Low battery warning resolved.
     case lowBatteryResolved
 
-    /// A software update is available.
```

Comment:
Should this be part of the diff? As far as I understand, available updates are announced via the methods declared in BluetoothReaderDelegate, which seem to have been removed as well.